packages/components/rollup.config.js

```diff
@@ -1,4 +1,15 @@
+import url from '@rollup/plugin-url';
+import svgr from '@svgr/rollup';
 import pkg from './package.json';
 import getConfig from '../../presets/react/rollup.config';
 
-export default getConfig(pkg, __dirname);
+const config = getConfig(pkg, __dirname);
+export default {
+  ...config,
+
+  plugins: [
+    ...config.plugins,
+    url(),
+    svgr({ svgo: false }),
```

Comment:
why no svgo, any issues?

Reply:
Didn't want to add logic for package specific svgo/svgr configs here. That's why I went with the most minimal setup for now.

Reply:
https://favro.com/organization/aee06c3fe6503c804f507dd9/22c630cc02b09f2d6f98b8b3?card=Goo-33024